domain/src/main/kotlin/no/nav/su/se/bakover/domain/satser/SatsFactoryForSupplerendeStønad.kt

```diff
@@ -33,84 +36,189 @@ class SatsFactoryForSupplerendeStønad(
         Grunnbeløpsendring(1.mai(2017), 1.mai(2017), 93634),
         Grunnbeløpsendring(1.mai(2018), 1.mai(2018), 96883),
         Grunnbeløpsendring(1.mai(2019), 1.mai(2019), 99858),
-        Grunnbeløpsendring(1.mai(2020), 1.mai(2020), 101351),
-        Grunnbeløpsendring(1.mai(2021), 1.mai(2021), 106399),
+        Grunnbeløpsendring(1.mai(2020), 4.september(2020), 101351),
+        Grunnbeløpsendring(1.mai(2021), 21.mai(2021), 106399),
         Grunnbeløpsendring(1.mai(2022), 20.mai(2022), 111477),
     ),
-) : SatsFactory {
-    private val grunnbeløpFactory: GrunnbeløpFactory = GrunnbeløpFactory(
-        clock = clock,
-        grunnbeløpsendringer = grunnbeløpsendringer,
-    )
-
-    private val minsteÅrligYtelseForUføretrygdede: MinsteÅrligYtelseForUføretrygdedeFactory =
-        MinsteÅrligYtelseForUføretrygdedeFactory.createFromFaktorer(
-            ordinær = listOf(
-                1.januar(2015) to Faktor(2.28),
-            ),
-            høy = listOf(
-                1.januar(2015) to Faktor(2.48),
-            ),
+    private val garantipensjonsendringerOrdinær: List<GarantipensjonFactory.Garantipensjonsendring> = listOf(
+        GarantipensjonFactory.Garantipensjonsendring(
+            1.januar(2016),
+            1.januar(2016),
+            162566,
+        ), // https://lovdata.no/forskrift/2015-11-20-1335/§1  kunngjort 23.11.2015 kl. 12.30 (Ikrafttredelse/virkningstidspunkt frem i tid til 01.01.2016 - dette er første innslag av garantipensjon)
+        GarantipensjonFactory.Garantipensjonsendring(
+            1.mai(2016),
+            1.mai(2016), 166274,
+        ), // https://lovdata.no/forskrift/2016-05-27-530/§5   kunngjort 30.05.2016 kl. 15.20 (Ikrafttredelse/virkningstidspunkt er begge tilbakevirkende til 01.05.2016)
+        GarantipensjonFactory.Garantipensjonsendring(
+            1.mai(2017),
+            1.mai(2017),
+            167196,
+        ), // https://lovdata.no/forskrift/2017-06-02-679/§5   kunngjort 02.06.2017 kl. 15.00 (Ikrafttredelse/virkningstidspunkt er begge tilbakevirkende til 01.05.2017)
+        GarantipensjonFactory.Garantipensjonsendring(
+            1.mai(2018),
+            1.mai(2018),
+            172002,
+        ), // https://lovdata.no/forskrift/2018-06-01-786/§5   kunngjort 01.06.2018 kl. 14.30 (Ikrafttredelse/virkningstidspunkt er begge tilbakevirkende til 01.05.2018)
+        GarantipensjonFactory.Garantipensjonsendring(
+            1.mai(2019),
+            1.mai(2019),
+            176099,
+        ), // https://lovdata.no/forskrift/2019-05-24-670/§5   kunngjort 24.05.2019 kl. 14.20 (Ikrafttredelse/virkningstidspunkt er begge tilbakevirkende til 01.05.2019)
```

Comment:
Fyller disser kommentarene et større mål enn å repetere verdiene som er oppgitt for ikrafttredelse/virkningstidspunkt?